README.md

```diff
@@ -60,6 +60,57 @@ export MARQUEZ_TIMEOUT_MS=5000
 
 ```
 
+### Common Calls and Examples
+There are two main ways to use the client:
+1. Set the namespace
+2. Supply the namespace for calls that require it
+
+#### Instantiating the client
+```mc = Client()```
+
+#### Set a Namespace for the client to use
+```mc.set_namespace(existing_namespace)```
```

Comment:
I don't think this is supported anymore. You either set the namespace in the constructor or you pass it in every call.
There is no such set_namespace()

Reply:
Good call. Adding more info about the ways to use the env vars or constructor to set.